src/components/i18nStrings.vue

```diff
@@ -135,11 +137,26 @@ export default {
 					this.$store.commit('cleanStateFor', this.path)
 				}
 			},
-      		deep: true
-		},
+			  deep: true,
+			  
+        	"$store.state.languages":function(languages){
```

Comment:
Add more spaces, like this. With our configuration, ESLint will complain if there's no space before {.
```suggestion
        	"$store.state.languages": function(languages) {
```
Check other lines in the commit as well to make the formatting more consistent with the existing code.